server/src/actors/project.rs

```diff
@@ -277,6 +281,9 @@ pub struct ProjectState {
     /// The organization id.
     #[serde(default)]
     pub organization_id: Option<u64>,
+    /// Random seed for jitter and related operations.
+    #[serde(skip, default = "project_state_seed")]
+    pub seed: f64,
```

Comment:
Is the intent that the dice are rolled once per project over the uptime of a relay, not once per fetch?

Reply:
The intent is once per fetch (as stated in the description):
> The seed changes with every fetch so that certain projects are not systematically discriminated.

... when Relay has long uptimes.